src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -19,26 +19,21 @@ public static class Warning
          public const string PopulationFileFormatIsNotSupported = "Population file format is not supported.";
          public const string InhibitorClearanceMustBeDefinedSeparately = "Please note that for the mechanism-based inactivator no clearance process is defined via the inactivation process by default. In theory, for every inactivated target molecule, also one inactivator molecule is cleared; this must be separately defined by the user in form of additional clearance processes for the inhibitor.";
          public const string FractionAbsorbedSmallerThanOne = "Absorption seems to be incomplete or absorption process is not finished. Vd, Vss (or Vd/F and Vss/F), t1/2, MRT and AUC_inf should be compared with respective PK-parameters from an IV simulation.";
+         public const string ExpressionParametersWillBeReset = "TODO: Expression parameter will be reset";
```

Comment:
@Yuri05 What should be the message here (this happens when the last entry in a localization category is deselected)